.travis.yml

```diff
@@ -5,11 +5,9 @@ language: ruby
 cache: bundler
 before_install: gem update bundler
 rvm:
-- 2.0
-- 2.1
-- 2.2
-- 2.3
```

Comment:
@fidalgo Thanks for your PR. Can you please add versions 2.0 - 2.3 back to the .travis.yml file. (We need to test and support these versions.)